src/stream.ts

```diff
@@ -49,7 +49,7 @@ export class Stream {
     this.name = name;
 
     let topic = this.name;
-    if (!this.isCollectionStream) {
+    if (!this.isCollectionStream && !this.name.includes("c8locals") && !this.name.includes("c8globals")) {
```

Comment:
Why we are doing this check in this pr ?

Reply:
Because if you are already passing `c8locals.stream` then it adds up `c8locals.c8locals.stream` like this. As you know we have done the same for the has a stream.